generators/form/index.js

```diff
@@ -1,39 +1,57 @@
 'use strict';
 const Generator = require('yeoman-generator');
 
+/**
+ * Helper that returns a date one year from today in the dumb standard USA style
+ * of M/D/YYYY
+ * 8/3/2019
+ */
+function getDate() {
+  const now = new Date();
+  const month = now.getMonth();
+  const date = now.getDate();
+  const year = now.getFullYear() + 1;
+  return `${month}/${date}/${year}`;
+}
+
 module.exports = class extends Generator {
   prompting() {
     const prompts = [
       {
         type: 'input',
         name: 'formNumber',
-        message: "What's your form number?"
+        message: "What's your form number?",
+        default: 'XX-230'
```

Comment:
Should we make this default more generic? such as XX-###?

Reply:
Maybe? Or also provide more sample values for this field?